source/mir/random/tinflex/internal/transformations.d

```diff
@@ -69,16 +77,19 @@ Table 1, column 5
 */
 S antiderivative(S)(in S x, in S c)
 {
-    import std.math : sgn;
-    import mir.internal.math : exp, log;
+    import std.math : copysign;
+    import mir.internal.math : exp, log, pow;
     if (c == 0)
         return exp(x);
     else if (c == -0.5)
         return -1 / x;
     else if (c == -1)
         return -log(-x);
+    auto d = c + 1;
+    if (c > 0)
+        return c / d * pow(x, d / c);
     else
-        return sgn(c) * c / (c + 1) * (sgn(c) * x)^^ ((c + 1) / c);
+        return - c / d * pow(-x, d / c);
```

Comment:
This can be simplified with as `fabs(c) / d * pow(x, d / c);`, pow will ignore sign